docker.go

```diff
@@ -296,19 +296,32 @@ func (c *DockerContainer) Stop(ctx context.Context, timeout *time.Duration) erro
 	return nil
 }
 
+// WithStopTimeout is a functional option that sets the timeout for the container termination.
+func WithStopTimeout(timeout time.Duration) TerminateOption {
+	return func(c *terminateOptions) {
+		c.stopTimeout = &timeout
+	}
+}
+
+// WithTerminateVolumes is a functional option that sets the volumes for the container termination.
+func WithTerminateVolumes(volumes ...string) TerminateOption {
+	return func(opts *terminateOptions) {
+		opts.volumes = volumes
+	}
+}
+
```

Comment:
bug: These are already covered by `StopTimeout` and `RemoveVolumes` in `cleanup.go`.
```suggestion
```